docs/integrations/builtin/core-nodes/n8n-nodes-base.manualworkflowtrigger.md

```diff
@@ -1,14 +1,32 @@
 ---
```

Comment:
Are these title and description formats something we want to update in the trigger template?

Reply:
Good call.